<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTypesEditor</class>
 <widget class="QDialog" name="CustomTypesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Types Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="customTypesLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="customTypesTable">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="typeDetailsLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="detailsTable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="toolTip">
    <string>Save Custom Props</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
   <property name="iconText">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load Button</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug...</string>
   </property>
   <property name="toolTip">
    <string>Debug</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
